calendar-webapp/src/main/webapp/javascript/eXo/calendar/EventTooltip.js

```diff
@@ -173,6 +173,9 @@
           html += '<div class="time clearfix"><div class="pull-left"><i class="'+className+'"></i></div><div class="text">' + info + '</div></div>';
         }
         if(data.location)    html += '<div class="location clearfix"><div class="pull-left"><i class="uiIconCalCheckinMini"></i></div><div class="text">' + data.location + '</div></div>';
+        var tmp = document.createElement("DIV");
+        tmp.innerHTML = data.description;
+        data.description = tmp.textContent || tmp.innerText || "";
```

Comment:
We could use Jquery for this : 
```suggestion
    gj(data.description).text();
```

Jquery Documentation : https://api.jquery.com/text/

Reply:
Upgrade Tomcat to 8.5.55

Reply:
Initial solution avoids HTML encoding issues.